### PR TITLE
feat(md-3356): native table images

### PR DESCRIPTION
**Pull request by gruskal**
Adds support for native table images. Allows multiple images set as the column representation, including sizing and positioning options.

### CODE REVIEW

File: android/src/main/java/com/qliktrialreactnativestraighttable/ClickableImageView.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+
+import java.util.Map;
+
+@SuppressLint("ViewConstructor")
+public class ClickableImageView extends androidx.appcompat.widget.AppCompatImageView implements SelectionsObserver {
```
Comment (vcellu) [inline android/src/main/java/com/qliktrialreactnativestraighttable/ClickableImageView.java:20]: Is there a reason why this couldn't be derived from ClickableText View?  Or sprouts a base class ClickableView?

Comment (gruskal) [reply android/src/main/java/com/qliktrialreactnativestraighttable/ClickableImageView.java:20]: Both of these implement SelectionsObserver, but other than that they're pretty different aren't they? One extends AppCompatImageView and the other extends AppCompatTextView

Comment (vcellu) [reply android/src/main/java/com/qliktrialreactnativestraighttable/ClickableImageView.java:20]: But we have code duplication here.  Is there something we can do about that?
`public  void onClear() {
    selected = false;
    updateBackgroundColor();
  }

  private void updateBackgroundColor() {
    int color = selected ? TableTheme.selectedBackground : Color.TRANSPARENT;
    int textColor = selected ? Color.WHITE : defaultTextColor;
    setBackgroundColor(color);
    setTextColor(textColor);
  }

  public void onRecycled() {
    if (cell.isDim) {
      selectionsEngine.remove(this);
    }
  }`

**Review by vcellu — APPROVED** [review]

**Pull request merged**